Dockerfile

```diff
@@ -1,9 +1,14 @@
-FROM alpine:3.8
+FROM golang:1.12
+WORKDIR /go/src/github.com/appvia/metal-pod-reaper/
+COPY . .
```

Comment:
Would require some restructuring of the repo, or a few `COPY` lines as you wouldn't want to add in everything by default (including readme, dot files, circleci etc.

Reply:
Actually not a big deal as you're using multi build process so it's not in the final image :)